packages/verification-vue-component/vue.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+    chainWebpack: config => {
+        config.module
+            .rule('js')
+            .exclude.add(/\.worker\.js$/)
+
+        config.module
+            .rule('worker')
+            .test(/\.worker\.js$/)
+            .use('worker-loader')
+            .loader('worker-loader')
+            .options({ filename: 'js/[name].[hash:8].js' })
+            .end()
```

Comment:
this worker stuff isn't needed if the only worker pdf.worker.js is never used, right?